Core/Main/elxParameterObject.h

```diff
@@ -64,7 +64,7 @@ class ParameterObject : public itk::DataObject
   void AddParameterMap( const ParameterMapType & parameterMap );
   const ParameterMapType& GetParameterMap( const unsigned int& index ) const;
   itkGetConstReferenceMacro( ParameterMap, ParameterMapVectorType );
-  unsigned int GetNumberOfParameterMaps() const { return this->m_ParameterMap.size(); }
+  unsigned int GetNumberOfParameterMaps() const { return static_cast< unsigned int >( this->m_ParameterMap.size() ); }
```

Comment:
Thanks for addressing this warning, Matt! I would have preferred changing the return type to `size_t` (instead of a brute-force `static_cast`),  but it appears that that changing the return type would affect other code. So as a quick fix, it seems acceptable to me. 👍